src/Adapters/Storage/Postgres/PostgresStorageAdapter.js

```diff
@@ -58,7 +58,8 @@ const toPostgresValue = value => {
 }
 
 const transformValue = value => {
-  if (value.__type === 'Pointer') {
+  if (typeof value == 'object' &&
```

Comment:
may as well go `===` for [consistency](https://www.goodreads.com/quotes/353571-a-foolish-consistency-is-the-hobgoblin-of-little-minds-adored) :)